bulk-ingester/src/main/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngesterConfig.java

```diff
@@ -23,18 +23,23 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  * #L%
  */
-package org.alfresco.hxi_connector.live_ingester;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
+package org.alfresco.hxi_connector.bulk_ingester.processor;
 
-@SpringBootApplication
-@SuppressWarnings("PMD.UseUtilityClass")
-public class BulkIngesterApplication
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Data
```

Comment:
WDYT about getting rid of `get` prefix and making the accessors fluent? Also do we need setters here?

Reply:
hmm, making getters fluent: +, but I think we have setters because it cannot fill properties without them. I saw in your PR you used records - so I maybe will try to also use records? imo it was really nice idea

Reply:
Spring can populate the values by constructor using so called `@ConstructorBinding` - you don't have to even annotate the class with that, just make the fields `final`

Reply:
I'll read about it - for now I used record and imo it looks good ;)